generators/generator-base.js

```diff
@@ -41,7 +41,7 @@ util.inherits(Generator, yeoman.Base);
  */
 Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enableTranslation) {
     try {
-        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.html';
+        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.component.html';
```

Comment:
The idea is to support generation of both ng1 and ng2 apps. so there will be question to choose version and based on that either client-1 or client-2 will be used. client 1 is ng1 as is today and wouldnt require file renames. client-2 is ng2 and hence the common methods like these will have an if else or something to support both.